showcase/tests/integration/components/hds/reveal/index-test.js

```diff
@@ -30,6 +30,20 @@ module('Integration | Component | hds/reveal/index', function (hooks) {
     assert.dom('.hds-reveal__toggle-button').hasText('More options');
   });
 
+  test('it provides the correct accessible name and description when passed in @text and @ariaDescribedBy content in the toggle button', async function (assert) {
+    await render(
+      hbs`
+        <h2 id="reveal-description">GitHub apps</h2>
```

Comment:
[nit] for consistency with the showcase example
```suggestion
        <span id="reveal-description">GitHub apps</span>
```